src/resdk/resources/annotations.py

```diff
@@ -115,6 +117,12 @@ def __init__(self, resolwe: "Resolwe", **model_data):
         self._sample: Optional[Sample] = None
         super().__init__(resolwe, **model_data)
 
+    @property
+    @assert_object_exists
+    def modified(self):
+        """Modification time."""
+        return parse_resolwe_datetime(self._original_values["modified"])
```

Comment:
Just a remark: I believe Resolwe uses ISO 8601 datetime. So you could just use `datetime.fromisoformat(date_string)` to convert from it to the `datetime` object.

Reply:
OK, will check this solve in one of the subsequent PRs.